_includes/v22.2/start-in-docker/mac-linux-steps-single-node.md

```diff
@@ -0,0 +1,133 @@
+When you use the `cockroach start-single-node` command to start a single-node cluster with Docker, additional features are available to help with testing and development.
+
+{{site.data.alerts.callout_danger}}
+Single-node clusters are not highly available or fault tolerant. They are not appropriate for production use.
```

Comment:
```suggestion
Single-node clusters are not highly available or fault-tolerant. They are not appropriate for production use.
```

Reply:
This is funny because Stephanie gave me the exact opposite feedback that resulted in no hyphen here. I can put it back.